modules/examples/health-check/health-client/src/main/scala/higherkindness/mu/rpc/healthcheck/client/monix/gclientMonix.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017-2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu.rpc.healthcheck.client.monix
+
+import cats.effect.{IO, Resource}
+import higherkindness.mu.rpc.ChannelFor
+import higherkindness.mu.rpc.config.channel.ConfigForAddress
+import higherkindness.mu.rpc.healthcheck.CommonRuntimeMonix
+import higherkindness.mu.rpc.healthcheck.serviceMonix.HealthCheckServiceMonix
+import org.log4s.getLogger
+
+object gclientMonix {
+
+  trait Implicits extends CommonRuntimeMonix {
+
+    val channelFor: ChannelFor =
+      ConfigForAddress[IO]("localhost", "50051").unsafeRunSync()
+
+    implicit val logger = getLogger
```

Comment:
log4cats?

Reply:
Yes, I told Juan Pedro before, I used log4s because other modules like client-cache or examples use it. Maybe we should make a full change (or maybe I should start with this...).

Reply:
I'll change from `log4s` to `log4cats`